src/app/extensions/quickorder/shared/quickorder-csv-form/quickorder-csv-form.component.html

```diff
@@ -25,6 +25,13 @@ <h3>{{ 'quickorder.page.csv.title' | translate }}</h3>
           >
           <small *ngSwitchDefault></small>
         </div>
+        <p>
```

Comment:
Wrong styling according to design example. 
![image](https://user-images.githubusercontent.com/50741106/136182643-6ea86107-2255-4aa4-b738-c581a42edd18.png)

Should be `<small class="form-text">` instead of `<p>`.

Reply:
Something needs to be done with the styling of the feedback message after a file was uploaded.
![image](https://user-images.githubusercontent.com/50741106/136185322-89e4bc9b-f389-4558-a866-57267589e591.png)